paddle/memory/CMakeLists.txt

```diff
@@ -1,6 +1,7 @@
 add_subdirectory(detail)
 
 cc_library(memory SRCS memory.cc)
+cc_library(memcpy SRCS memcpy.cc)
```

Comment:
That's great to separate memory and memcpy.

But, shall we put `memcpy` into `paddle_memory`?  That's convenient to be used outside.